lib/h3core.js

```diff
@@ -606,6 +622,20 @@ export function h3ToChildren(h3Index, res) {
     return out;
 }
 
+/**
+ * Get the center child of the given hexagon at a particular resolution
+ * @static
+ * @param  {H3Index} h3Index  H3 index to get center child for
+ * @param  {Number} res       Resolution of hexagon to return
+ * @return {H3Index}          H3 index of child, or null for invalid input
+ */
+export function h3ToCenterChild(h3Index, res) {
+    validateIndex(h3Index);
+    validateRes(res);
+    const [lower, upper] = h3IndexToSplitLong(h3Index);
+    return readH3Index(H3.h3ToCenterChild(lower, upper, res));
```

Comment:
Any reason why these two lines aren't simply `return readH3Index(H3.h3ToCenterChild(...h3IndexToSplitLong(h3Index), res));`

If my comment above about not forcing the validation is also fine, the whole thing could be rewritten as:

```js
export const h3ToCenterChild = (h3Index, res) => readH3Index(H3.h3ToCenterChild(...h3IndexToSplitLong(h3Index), res))
```